jta-jpa/src/main/java/org/acme/CamelRoutes.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.acme;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.model.rest.RestParamType;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
+public class CamelRoutes extends RouteBuilder {
+    @ConfigProperty(name = "camel.servlet.mapping.context-path")
```

Comment:
Can we not just use `camel.rest.context-path` for this?

Reply:
OK, I just borrow these codes from https://github.com/apache/camel-spring-boot-examples/blob/main/spring-boot-jta-jpa/src/main/java/sample/camel/CamelRoutes.java

So is there any difference between these two property?

Reply:
`camel.servlet.mapping.context-path` is a component property of `camel-servlet`, and we are not using that component in this example.

So to avoid confusion, it's better to use `camel.rest.context-path` and then there will be no need for the `ConfigProperty` injection (I think).

Reply:
Oh, yeah, I get it.